README.md

```diff
@@ -40,16 +40,17 @@ Documentation projects have traditionally been part of a software project. Howev
 
 ## Sphinx projects
 
-- [Scrapy](https://docs.scrapy.org/en/latest/) - Embeds a lot of reference snippets and uses `sphinx-hoverxref` for quick reference tooltips. Lots of inspiration to be found in content organization. **#sphinx** **#sphinx-hoverxref**.
-- [setuptools](https://setuptools.pypa.io/en/latest/) - lots of features, using the Furo theme. [Twitter thread](https://twitter.com/readthedocs/status/1546527820150718469) with some examples. **#sphinx** **#sphinx-themes**.
-- [sphinxcontrib-needs](https://sphinxcontrib-needs.readthedocs.io/en/latest/) - Documentation of `sphinxcontrib-needs`. **#sphinx** **#sphinx-themes** **#ui-material**.
-- [sphinx-immaterial](https://sphinx-immaterial.readthedocs.io/en/latest/) - Documentation of `sphinx-immaterial`, a Material theme for Sphinx, based on Material for MkDocs. **#sphinx** **#sphinx-themes** **#ui-material**.
+- [Scrapy](https://docs.scrapy.org/) - Embeds a lot of reference snippets and uses `sphinx-hoverxref` for quick reference tooltips. Lots of inspiration to be found in content organization. **#sphinx** **#sphinx-hoverxref**.
+- [setuptools](https://setuptools.pypa.io/) - lots of features, using the Furo theme. [Twitter thread](https://twitter.com/readthedocs/status/1546527820150718469) with some examples. **#sphinx** **#sphinx-themes**.
+- [sphinxcontrib-needs](https://sphinxcontrib-needs.readthedocs.io/) - Documentation of `sphinxcontrib-needs`. **#sphinx** **#sphinx-themes** **#ui-material**.
+- [sphinx-immaterial](https://sphinx-immaterial.readthedocs.io/) - Documentation of `sphinx-immaterial`, a Material theme for Sphinx, based on Material for MkDocs. **#sphinx** **#sphinx-themes** **#ui-material**.
 - [Uberspace](https://manual.uberspace.de/) - Customized sidebar and footer, adding project's branding through custom CSS and HTML to `sphinx_rtd_theme`. Latest version and release date on front page. **#sphinx** **#sphinx-themes** **#custom-theme**.
-- [Weblate](https://docs.weblate.org/en/latest/) - Weblate is a translation platform with a large documentation project with many translations and customized Read the Docs theme. Documentation aimed at all segments: users, administrators and developers. Also features an extensive Changelog. **#sphinx** **#sphinx-themes** **#translation**.
+- [Wagtail](https://docs.wagtail.org/) - Wagtail is a Django-based CMS with a global community, the documentation spans multiple stakeholders (editors and developers), has it's own beautiful theme and is largely structured around Diátaxis ideals. The [Release Notes](https://docs.wagtail.org/en/stable/releases/index.html) and [Contribution guide](https://docs.wagtail.org/en/stable/contributing/index.html) are remarkable. Project uses a minimal set of Sphinx extensions. **#sphinx** **#sphinx-themes** **#diataxis**.
```

Comment:
CC: @humitos noted a few things about the Wagtail Documentation here

I think the theme, the simplicity of the Sphinx setup, the organization of content and the huge number of contributors is where this project stands out.

Screenshot-wise, the editor guide has some pretty pages: https://docs.wagtail.org/en/stable/editor_manual/getting_started.html

I was curious about how many people have written the docs, so here comes a tip perhaps interesting to the update:

Wagtail currently has 579 contributors. The docs/ folder has 514 contributors, of which 348 contributors have taken part in the release notes -- hosting your release notes inside the docs is great for readers but can also function as a good way to make developers discover how to contribute to the documentation while they are busy with the code.

Here are some quick figures from their contribution list:

```
➜  wagtail git:(main) git shortlog -n -s -- docs | wc -l
514
➜  wagtail git:(main) git shortlog -n -s -- docs/releases/ | wc -l
348
➜  wagtail git:(main) git shortlog -n -s -- docs/advanced_topics/ | wc -l
138
➜  wagtail git:(main) git shortlog -n -s -- docs/getting_started/ | wc -l
41
➜  wagtail git:(main) git shortlog -n -s -- docs/editor_manual/ | wc -l
46
➜  wagtail git:(main) git shortlog -n -s -- docs/extending/ | wc -l
22
➜  wagtail git:(main) git shortlog -n -s -- docs/contributing/ | wc -l
46
➜  wagtail git:(main) git shortlog -n -s -- docs/reference/ | wc -l
159
➜  wagtail git:(main) git shortlog -n -s -- docs/topics/ | wc -l
95
```